dcos/cosmospackage.py

```diff
@@ -582,9 +583,8 @@ def _get_header(request_type, version):
     :returns: header information
     :rtype: str
     """
-
-    if request_type == 'add-response':
-        return 'application/vnd.dcos.universe.package+json;charset=utf-8;version=v3'
+    if request_type == 'add-request':
```

Comment:
FYI, this won't work when we have to implement universe install. There we are going to use `add-request`.